core/tests/performance_framework/perf_services.py

```diff
@@ -355,20 +418,45 @@ def _create_exploration(self, driver):
         return new_exploration_url
 
     def _setup_login(self, driver):
+        """Sets up user login
```

Comment:
Add a fullstop here -> `Sets up user login.`

Reply:
Thanks for the careful observation!
Done!